api/backend_test.go

```diff
@@ -386,6 +386,39 @@ func TestStartStopMultipleTimes(t *testing.T) {
 	require.NoError(t, backend.StopNode())
 }
 
+func TestSignHash(t *testing.T) {
+	backend := NewStatusBackend()
+	config, err := utils.MakeTestNodeConfig(params.StatusChainNetworkID)
+	require.NoError(t, err)
+
+	require.NoError(t, backend.StartNode(config))
+	defer func() {
+		require.NoError(t, backend.StopNode())
+	}()
+
+	testSignature := func(chatPrivKeyHex, hashHex, expectedSignatureHex string) {
```

Comment:
What's the reason of this? Usually a table driven approach is expected.

Reply:
It is kind of table driven, but there is only one item to test... but it is easy to extend it later.

Reply:
adressed